src/transformers/onnx/convert.py

```diff
@@ -112,19 +112,34 @@ def export(
 
         config.patch_ops()
 
-        # export can works with named args but the dict containing named args as to be last element of the args tuple
-        export(
-            model,
-            (model_inputs,),
-            f=output.as_posix(),
-            input_names=list(config.inputs.keys()),
-            output_names=onnx_outputs,
-            dynamic_axes={name: axes for name, axes in chain(config.inputs.items(), config.outputs.items())},
-            do_constant_folding=True,
-            use_external_data_format=config.use_external_data_format(model.num_parameters()),
-            enable_onnx_checker=True,
-            opset_version=opset,
-        )
+        # PyTorch deprecated the `enable_onnx_checker` and `use_external_data_format` arguments in v1.11,
+        # so we check the torch version for backwards compatibility
+        if parse(torch.__version__) < parse("1.11.0.dev0"):
```

Comment:
I found that I had to include the `dev0` part of the version number to ensure the conditional evaluates to `False` for the nightly version of `torch`. Is this OK until v1.11 is released?

Reply:
Let's do <= 1.10.99 (don't think we will reach that micro ;-) ) otherwise.

Reply:
Nice idea! Will include it :)